pages/repo/edit.js

```diff
@@ -438,11 +460,29 @@ export class EditorPage extends Component {
     const isNew = commitId === 'new'
     let committedEditorState
     if (isNew) {
-      committedEditorState = this.editor.newDocument(
-        router.query,
-        this.props.me
-      )
-      debug('loadState', 'new document', committedEditorState)
+      if (templateRepo) {
+        const commit = templateRepo.latestCommit
+        const json = {
+          ...commit.document.content,
+          // add format & section to root mdast node
+          format: commit.document.meta.format,
+          section: commit.document.meta.section
+        }
+
+        const title = findTitleLeaf(json)
+        title.value = router.query.title
+
+        json.meta.auto = true
```

Comment:
Let's save a reference to the template for later use:
```suggestion
        json.meta.auto = true
        json.meta.templateRepoId = router.query.templateRepoId
```